src/ontology/external/mondo-omim-genes.robot.tsv

```diff
@@ -1166,7 +1164,6 @@ http://purl.obolibrary.org/obo/MONDO_0010430	http://identifiers.org/hgnc/13128	O
 http://purl.obolibrary.org/obo/MONDO_0010431	http://identifiers.org/hgnc/2567	OMIM:300804	https://omim.org/entry/300170
 http://purl.obolibrary.org/obo/MONDO_0010435	http://identifiers.org/hgnc/20145	OMIM:300814	https://omim.org/entry/300808
 http://purl.obolibrary.org/obo/MONDO_0010437	http://identifiers.org/hgnc/8768	OMIM:300816	https://omim.org/entry/300169
-http://purl.obolibrary.org/obo/MONDO_0010438	http://identifiers.org/hgnc/8957	OMIM:300818	https://omim.org/entry/311770
```

Comment:
This is correctly removed based on https://github.com/monarch-initiative/omim/blob/main/data/exclusions-disease-gene.tsv#L6